pypeit/core/procimg.py

```diff
@@ -979,13 +951,13 @@ def variance_frame(datasec_img, sciframe, gain, ronoise, darkcurr=None,
     # ToDO JFH: I would just add the darkcurrent here into the effective read noise image
     # The effective read noise (variance image)
     if rnoise is None:
-        rnoise = rn_frame(datasec_img, gain, ronoise)
+        rnoise = rn2_frame(datasec_img, gain, ronoise)
 
     # No sky frame provided
     if skyframe is None:
         _darkcurr = 0 if darkcurr is None else darkcurr
         if exptime is not None:
-            _darkcurr *= exptime/3600.
+            _darkcurr *= exptime #/3600.
         var = np.abs(sciframe - np.sqrt(2.0)*np.sqrt(rnoise)) + rnoise + _darkcurr
         var = var + adderr**2*(np.abs(sciframe))**2
```

Comment:
I think we could be using the utils.sn_clip function here which would at least streamline this other similar noise floors applied elsewhere in the pipeline.

Reply:
Do you mean `utils.clip_ivar` ?

Reply:
Yes.